### DISCUSSION TITLE
Ex-Falso: How to embed in image?

### DISCUSSION THREAD

**Discussion by octimis** [quodlibet/quodlibet#4265] [Q&A]
My OS is Ubuntu 22.

I enabled the plug in 'Edit Embedded Image'

I right click on the file in the 'Songs Pane', and successfully remove the image with right click > edit embedded image >  remove all images.
But nothing happens I select 'Embed current image', even when its copied to clipboard.

Edit: It seems the image file must named 'cover' or 'front' and be located in the same directory as the track, for embed image to work. It also seems, Nautilus file explorer's fault (not Ex-falso) for the thumbnail not showing, as other programs like Dolphin and media players could display the art work.

**Comment by declension** [comment]
You can now choose a file on disk to embed for this kind of use-case. Hope that helps
